test/e2e/specs/specs-calypso/wp-manage-domains-spec.js

```diff
@@ -187,16 +187,16 @@ describe( `[${ host }] Managing Domains: (${ screenSize }) @parallel`, function
 			return await enterADomainComponent.enterADomain( blogName );
 		} );
 
-		it( 'Can add domain to the cart', async function () {
+		it.skip( 'Can add domain to the cart', async function () {
```

Comment:
Do you think we can delete these tests instead of disabling them?

Reply:
Yep, I can do that. When I was working on the PR I wasn't sure and wanted to deploy it so decided to disable them. But now that I think about it - there is no sense of keeping those. I'll prepare a PR